### DISCUSSION TITLE
Need clarification on Licensing.

### DISCUSSION THREAD

**Discussion by sabinayakc** [toeverything/AFFiNE#6646] [Q&A]
Hello everyone,

I'm interested in forking this repo to make some front end modifications and deploy it as a Software as a Service (SaaS) application on my own server. I'd like to potentially allow other users to use this modified version.

Could someone clarify what I can and cannot do with the original codebase under its current license? Additionally, what kind of licensing agreement should I consider to bind by for this type of usage?

Can I just use the front end peace, and make my own backend? I am little confused. 

Thank you in advance for your guidance!

**Comment by EYHN** [comment]
All code except the backend is under the MIT license, you can make any changes you want. The backend code is also MIT for personal use, but you can't use it for busniess.

**Comment by forehalo** [comment]
selfhost user now granted Pro quota as it on affine.pro

you may roll up your image and do a redeploy to make to work
